src/board_controller/openbci/ganglion_bglib/callbacks.cpp

```diff
@@ -93,7 +93,7 @@ void ble_evt_connection_disconnected (const struct ble_msg_connection_disconnect
 {
     // atempt to reconnect
     ble_cmd_gap_connect_direct (
-        &GanglionLib::connect_addr, gap_address_type_random, 40, 60, 100, 0);
+        &GanglionLib::connect_addr, gap_address_type_random, 10, 76, 100, 0);
```

Comment:
https://github.com/tudelft/bluegiga/blob/master/Documentation/Bluetooth_Smart_Software_v1.3.1_API_Reference.pdf


Page 98

![Screen Shot 2020-06-01 at 4 58 39 PM](https://user-images.githubusercontent.com/25914123/83459000-259ecd00-a429-11ea-94f5-23aef9b2bc3f.png)

Reply:
yep but it was 0 even before, I still dont get how connection interval affects packet loss and why this connection has a timeout in general

Reply:
I guess as a "safety net" for when no data has been received. Example: When walking out of range, eventually the BLED connection may close after the `timeout` is reached.

In contrast, Cyton and Dongle will always keep transmitting/listening, and will re-acquire the data stream as soon as they are in range.

This is from my experience testing the Cyton and Ganglion connections and packet loss.

Reply:
Or they may both re-connect. It's been a while.